lib/main.coffee

```diff
@@ -351,10 +351,10 @@ module.exports = Hydrogen =
             @wsKernelPicker = new WSKernelPicker (kernel) =>
                 @clearResultBubbles()
 
-                grammar = @editor.getGrammar()
+                grammar = kernel.grammar
```

Comment:
As far as I can see from the code, `kernel.grammar` is set to the value of `editor.getGrammar()` at the time when the kernel is constructed (following the call chain from `main.coffee::createResultBubble`)

There is a separate `language` field, but I don't believe that it is in play here.